addons/last-edit-tooltip/userscript.js

```diff
@@ -0,0 +1,8 @@
+export default async function ({ addon, global, console }) {
+  fetch("https://api.scratch.mit.edu" + document.location.pathname).then(function(response) {
```

Comment:
The current implementation looks like is not going to work if the user goes inside the editor, then outside. Please do something like this after getting the information from the API:
```js
while(true) {
  const element = await addon.tab.waitForElement(".share-date", {markAsSeen:true});
  // lines 5 and 6 - you can use "element" instead of the query selector
}
`

Reply:
This is giving me an error saying: `Uncaught (in promise) SyntaxError: Unexpected reserved word` and this is the code I have in it:

 
```js
export default async function ({ addon, global, console }) {
  fetch("https://api.scratch.mit.edu" + document.location.pathname).then(function(response) {
    return response.json()
  }).then(function(text) {
    while(true) {
      const element = await addon.tab.waitForElement(".share-date", {markAsSeen:true});
      let dateMod = new Date(text.history.modified)
      element.setAttribute("title", `Modified: ${dateMod.toLocaleString('en-us',{month:'short'})} ${dateMod.getDay()}, ${dateMod.getFullYear()}`)
    }
  });
}

```

Reply:
The error is coming from the `const element = await addon.tab.waitForElement(".share-date", {markAsSeen:true});` line.